tests/models/test_playerscore.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+from django.db.utils import IntegrityError
+
+from mars_api.models import Game, Player, PlayerScore
+
+pytestmark = pytest.mark.django_db
+
+
+def test_can_create_multiple_player_scores_same_player(player, player_score_factory):
+    ps1 = player_score_factory(player=player)
+    ps2 = player_score_factory(player=player)
+
+    assert PlayerScore.objects.filter(player=player).count() == 2
+
+
+def test_player_not_allowed_to_have_multiple_scores_in_the_same_game(
+    game, player_score_factory
+):
+    """
+    Tests if only unique players can be added to the PlayerScore with same game_id.
+    (So that there wouldn't be a player that has two scores for the same Game.)
```

Comment:
**So that there isn't a player** I thiiink

Reply:
![image](https://user-images.githubusercontent.com/34490649/92322485-5e060680-f029-11ea-8ee0-1da46368063d.png)
Interesting observation :monocle_face: 
"Possible result of an imagined condition" we imagine in the future that that there will never be a case where a player has two scores for the same Game. While testing, we are thinking about the things that can go wrong, but in reality they won't because we checked them before?
On the contrary, using present simple is about *now*, kind of feels like we are currently adding `player_scores` while testing.

Reply:
Yeah, so I assumed you're using the first conditional: _Tests if only unique players **can** ~could~ be added_

Reply:
aah, line 20, not 21